.config/tmux/tmux.conf

```
@@ -50,7 +50,7 @@ set -g mouse on
 set -g base-index 1
 setw -g pane-base-index 1
 
-bind r source-file ~/.tmux.conf \; \
+bind r source-file ~/.config/tmux/tmux.conf \; \
```

```suggestion
bind r source-file $XDG_CONFIG_HOME/tmux/tmux.conf \; \
```
if this env will work in .tmux.conf

---

tested, and this works